### ISSUE TITLE
File Marked as Uncommon or Unsafe

### ISSUE THREAD

**Issue by avin-6998** [Abe404/root_painter#147]
Hello,

I downloaded the RootPainterInstaller 0.2.27 Windows.exe file from your website, and my browser/Windows SmartScreen displayed a warning stating:
"This file is not commonly downloaded and could be unsafe."

I trust your website, but this warning may deter other users from proceeding with the download. I believe this could be due to the file not being widely distributed or missing a digital signature

**Comment by Abe404** [comment]
I agree - unfortunately application signing is not very straightforward but it should be possible.

**Comment by azarbarzin-avin** [comment]
Is it related to the connection with the server? I have tried many ways, but unfortunately, I could not solve it.

**Comment by Abe404** [comment]
This is when you install the client right? I think the problem is just that it's an exe that is not signed by Microsoft. I don't think it's anything to do with the server connection.

**Comment by Abe404** [comment]
@avin-6998 Are you saying you cannot install it? You should be able to install it anyway even though there is a warning.
